framework/helpers/preferences/preferences.livecodescript

```diff
@@ -369,64 +369,37 @@ function appGetPreferenceFile pUserOrShared
   put prefFilenameForPlatform(pUserOrShared) into tFile
 
   if tFile is not empty then
-    if pUserOrShared is "user" then
-      return _systemPreferencesFolder() & slash & tFile
-    else
-      return _systemAppDataFolder("shared") & slash & tFile
-    end if
+    return _preferencesFolder(pUserOrShared) & slash & tFile
```

Comment:
Please change `slash` to `"/"` as that is the standard used elsewhere in the code.